libs/lib-client/diagram/src/lib/diagram/diagram.component.html

```diff
@@ -4,6 +4,7 @@
     *ngFor="let row of tileMatrix; trackBy: trackRowBy; let rowIndex = index"
     @diagramRow
     [row]="row"
+    [rowParents]="rowParentsList[rowIndex]"
```

Comment:
If you can move the lookup information to the service we can ignore this

Reply:
updated the code, now consuming required data from service instead of an input property bind 👍